unity-environment/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgentConfigurable.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Collections;
+using System.Collections;
```

Comment:
There does not seem to be any difference.

Reply:
Usually this is due to line ending difference.